lesson-3/src/main/java/lesson3/controllers/HelloController.java

```diff
@@ -10,6 +10,12 @@
 
     @GetMapping
     public String getHello() {
-        return "Hello!!!";
+        String message = null;
```

Comment:
\[Checkstyle\] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4\.